### ISSUE TITLE
library not found for -lPods when I mix Swift and ObjC pods

### ISSUE THREAD

**Issue by georgbachmann** [CocoaPods/CocoaPods#3143]
Hello,
in a project that I am working on, I want to use some swift pods and some non-swift pods.
So I added GPUImage and tried using it but when compiling i get `ld: library not found for -lPods`
I already did clean and even delete all pods (incl. the project file) and made a new pod install. I have 0.36.0.beta.2 of cocoapods installed...

Is this a known issue? Or is there a workaround?

Here's my pod file:

    source 'https://github.com/CocoaPods/Specs.git'
    platform :ios, '8.0'

    pod 'Alamofire', '~> 1.1'
    pod 'GPUImage', '~> 0.1.6'

**Comment by DanielRakh** [comment]
Having a similar issue when importing pop by Facebook. Any ideas about how to resolve this?

**Comment by georgbachmann** [comment]
I don't know.. I ended up not using Cocoapods at the moment... I had too many issues when f.e. building for Instruments, and so on... Hopefully that will all get better in the future!
But if you find any solution I'd be super happy to learn about it!!!

**Comment by DanielRakh** [comment]
Will do. Thanks!

**Comment by kylef** [comment]
This error indicates previously integrated state for static library pods that should be removed when you `pod install` after adding the Swift pod and the integration is converted to frameworks.

I'd suggest using https://github.com/kylef/cocoapods-deintegrate to deintegrate to remove this broken state and then run `pod install` again.

**Comment by kylef** [comment]
I've tried to reproduce this issue but I can't seem to do it. Have you manually added any pod library references to your project?

**Comment by georgbachmann** [comment]
Hey Kylef,
no, I first added

    pod 'Alamofire', '~> 1.1'

then continued with my project. After a while I needed GPUImage, so I added

    pod 'GPUImage', '~> 0.1.6'


made a pod update and then it wasn't working any more. I tried removing all cocoapods (not with your tool, but my deleting the workspace, pod.lock and pod directory) and tried a new pod install. Didn't work either. When removing GPUImage again it worked again...
